geometry_msgs/Vector3[] goals
---
bool success
string message
